src/components/governance/GovernanceStats.jsx

```diff
@@ -60,7 +60,7 @@ export default function GovernaceStats() {
               />
             </Grid.Col>
             <Grid.Col xs={6}>
-              {protocol.borrowRatePerBlock && (
+              {chain && protocol.borrowRatePerBlock && (
```

Comment:
Instead just destructure `borrowRatePerBlock` from `protocol`  on line 29.

```
const {
    totalStaked = ZERO,
    totalBorrows = ZERO,
    getLoanableAmount = ZERO,
    borrowRatePerBlock = ZERO
  } = protocol;
```

Reply:
I've destructured `borrowRatePerBlock` so things are cleaner but we still need the double conditional as `ProtocolData.jsx` defaults to mainnet when disconnected so `borrowRatePerBlock` has a truthy value